public/blog-posts/liveness-2-and-beyond.md

```diff
@@ -10,7 +10,7 @@ Let’s first recap the problem we are trying to solve with the Liveness Module.
 
 Unlike EOAs, smart wallets have configuration settings, which can cause synchronization problems across chains. Consequently, SAFEs on different chains function as separate contracts, even though they may share the same address and configuration parameters during deployment. This problem becomes critical when there’s a change in the owners’ list. We must tackle this issue if we want to create a chain-agnostic future.
```

Comment:
Also, the preview is not showing the images

Reply:
fixed https://github.com/defi-wonderland/web/pull/58/commits/7acb5b37855600479acb8fba03f0c6dba30805b6